ui/packages/app/src/components/Audio/index.ts

```diff
@@ -2,8 +2,15 @@ export { default as Component } from "./Audio.svelte";
 export const modes = ["static", "dynamic"];
 
 export const document = () => ({
-	type: "{ name: string; data: string }",
-	description: "audio data as base64 string",
+	type: {
+		input_payload: "{ name: string; data: string }",
+		response_object: "{ name: string; data: string, is_file: boolean }"
+	},
+	description: {
+		input_payload: "audio data as object with filename and base64 string",
+		response_object:
+			"audio data as object with filename. `{DEMO URL}/file={name}` hosts the audio"
```

Comment:
I think we can reword this to make it a little clearer:
```suggestion
			"object that includes path to audio file. Can be accessed at the URL: `{DEMO URL}/file={name}`"
```
Would it be possible by any chance to replace {DEMO URL} with the actual URL of the demo? That would be pretty cool

Reply:
This is knocking around somewhere. Maybe `base_api`, which we pass to the API, would do the trick.

Reply:
Reworded it and replaced with actual URL 


https://user-images.githubusercontent.com/9021060/220811737-af7ccf7c-97b8-48ec-b5e5-28a0cf506877.mov